core/txbuilder/finalize_test.go

```diff
@@ -25,6 +25,69 @@ import (
 	"chain/testutil"
 )
 
+func TestSighashCheck(t *testing.T) {
```

Comment:
is it possible to have this test in txbuilder so we don't need to export anything we wouldn't otherwise?